addons/web/static/src/views/fields/badge_selection/badge_selection_field.js

```diff
@@ -116,11 +117,19 @@ export const badgeSelectionField = {
             ],
             default: "md",
         },
+        {
+            label: "Inline",
```

Comment:
I find "Inline" unclear. I know that you're referring to the desired visual result ("all badges in one line"), but strictly HTML speaking, inline content could wrap on a second line, so it could be misleading.
I'd reword it to to something else, "Nowrap" for example